hadoop-hdds/common/src/main/proto/StorageContainerLocationProtocol.proto

```diff
@@ -106,7 +106,7 @@ enum Type {
   ListContainer = 4;
   DeleteContainer = 5;
   QueryNode = 6;
-  NotifyObjectStageChange = 7;
+  CloseContainer = 7;
```

Comment:
We should stop doing with after the first Beta. These things break backward compact. Not relevant for this change.

Reply:
Agreed.